### PR TITLE
Duplicate mine filter for each enhet and change endpoints to consider enhet id

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
   test-build-and-push:
     name: Test, build and push
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/master'
+    if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/master' || github.ref == 'refs/heads/CleanupMineFilter'
```
Comment (stianStensli) [inline .github/workflows/main.yaml:30]: Dette kan kanskje fjernes når før det går til dev?

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -52,7 +52,7 @@ jobs:
           docker push ${IMAGE}:${IMAGE_TAG}
   deploy-q0:
     name: Deploy application to Q0
-    if: github.ref == 'refs/heads/dev'
+    if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/CleanupMineFilter'
```
Comment (stianStensli) [inline .github/workflows/main.yaml:55]: samme som over

File: src/main/no/nav/pto/veilarbfilter/routes/MineLagredeFilterRoutes.kt [hunk 3]
```diff
@@ -32,27 +32,36 @@ fun Route.conditionalAuthenticate(useAuthentication: Boolean, build: Route.() ->
     return route
 }
 
-fun Route.mineLagredeFilterRoutes(mineLagredeFilterService: FilterService, useAuthentication: Boolean) {
+fun Route.mineLagredeFilterRoutes(mineLagredeFilterService: MineLagredeFilterServiceImpl, useAuthentication: Boolean) {
     conditionalAuthenticate(useAuthentication) {
-        route("/minelagredefilter") {
+        route("/minelagredefilter/{enhetId}/") {
```
Comment (stianStensli) [inline src/main/no/nav/pto/veilarbfilter/routes/MineLagredeFilterRoutes.kt:37]: This means that it is important to also update all applications that uses veilarbfilter?

**Review by stianStensli — CHANGES_REQUESTED** [review]
If I understood it correctly then this PR will cause some down-time, for the applications using veilarbfilter

File: .github/workflows/main.yaml (hunk 1 above)
Comment (dragutin-nav) [reply .github/workflows/main.yaml:30]: yep, I agree...it was just temporary solution to test changes.

File: .github/workflows/main.yaml (hunk 2 above)
Comment (dragutin-nav) [reply .github/workflows/main.yaml:55]: I agree.

File: src/main/no/nav/pto/veilarbfilter/routes/MineLagredeFilterRoutes.kt (hunk 3 above)
Comment (dragutin-nav) [reply src/main/no/nav/pto/veilarbfilter/routes/MineLagredeFilterRoutes.kt:37]: only our frontend application is using mine filter for now, so its safe to change little bit endpoind. I also updated frontend, so we will need to releaes both at similar time (probably outside working hours)

**Review by pederpus — APPROVED** [review]

**Comment by dragutin-nav** [comment]
Chaning logic for MineFilter cleanup. Its not necessary anymore to duplicate mine filter, but to connect veiledergruppe id with mine filter. So in case if veiledergruppe is deleted, we will delete mine filter. If veiledergruppe is updated, mine filter will be automatically synchronised with changes.

**Pull request closed without merging**